### ISSUE TITLE
[BUG] Grammatical errors in documentation

### ISSUE THREAD

**Issue by Saket2** [appsmithorg/appsmith#629]
Open https://docs.appsmith.com/

Note - 'Present' is what is already present on the website and 'Suggestion' is what I'm suggesting.

1. Introduction 

Present - Invest weeks of time building it from scratch and fighting with HTML/CSS
Suggestion - Investing weeks building it from scratch and fighting with HTML/CSS

Present - Getting started with an admin panel and invest even more time customising it
Suggestion - Getting started with an admin panel and invest even more time customizing it

2. Getting Started  

Present - MacOS
Suggestion - macOS

Present - Enable execute & root permissions. The script requires root permissions to install relevant packages on systems other than Mac OS
Suggestion - Enable execute & root permissions. The script requires root permissions to install relevant packages on systems other than macOS

Present - Ensure your security groups are configured to allow traffic to port 80 & 443 on your installation instance. 
Suggestion - Ensure your security groups are configured to allow traffic to ports 80 & 443 on your installation instance.

3. FAQs

Present - We sincerely hope this never happens to anyone but we acknowledge that there can be road blocks. At this juncture we're working on a way for you to be able to write custom react components and plugins to unblock you from any platform limitations. In the meantime, if absence of any feature is limiting you, you can write to support@appsmith.com or reach out to us on our discord server 
Suggestion - We sincerely hope this never happens to anyone but we acknowledge that there can be roadblocks. At this juncture, we're working on a way for you to be able to write custom react components and plugins to unblock you from any platform limitations. In the meantime, if an absence of any feature is limiting you, you can write to support@appsmith.com or reach out to us on our discord server 

Present - Appsmith isn't built for complicated UI interactions that are common to e-commerce or other consumer facing websites. It works great for tools internal to your team or organization but you wouldn't get very far trying to build an attractive consumer website. For those use cases, we'd recommend you try Wix, Squarespace or Webflow. You can also perform some simple data visualisation & charting on Appsmith but for more in-depth business intelligence & analytics, we'd recommend Tableau, Looker or Redash.
Suggestion - Appsmith isn't built for complicated UI interactions that are common to e-commerce or other consumer-facing websites. It works great for tools internal to your team or organization but you wouldn't get very far trying to build an attractive consumer website. For those use cases, we'd recommend you try Wix, Squarespace, or Webflow. You can also perform some simple data visualization & charting on Appsmith but for more in-depth business intelligence & analytics, we'd recommend Tableau, Looker, or Redash.

Present - Appsmith is the first open source visual framework for building internal tools. We make building internal tool building easy by giving you building blocks (widgets, APIs, Queries) which you can configure and connect together while the framework does most of the heavy lifting
Suggestion - Appsmith is the first open-source visual framework for building internal tools. We make building internal tool building easy by giving you building blocks (widgets, APIs, Queries) which you can configure and connect while the framework does most of the heavy lifting

4. Enabling Third Party Services

Present  - Our self hosted version allows you to configure your own keys to enable third party services such as Google OAuth. To enable a service, simply open the folder of your appsmith installation and edit the docker.env file.
Suggestion - Our self-hosted version allows you to configure your own keys to enable third party services such as Google OAuth. To enable a service, simply open the folder of your appsmith installation and edit the docker.env file.

5. Building The UI

Present  - Widget properties can be editing via the property pane which can be toggled next to the delete icon. Widget properties can affect data, style and action to take. 
Suggestion - Widget properties can be edited via the property pane which can be toggled next to the delete icon. Widget properties can affect data, style, and action to take. 

6. Displaying API / Query Data

Present  - You can use Javascript inside the {{ }} to transform API / Query data when binding it to a property. An example of this would be an API / Query which returns an array of values that are needed to populate a dropdown.
A dropdown needs an Array<label, value> in it's option field, so in order to connect this data to a dropdown, we need to transform the data in the dropdown options property
Suggestion - You can use Javascript inside the {{ }} to transform API / Query data when binding it to a property. An example of this would be an API / Query which returns an array of values that are needed to populate a dropdown.
A dropdown needs an Array<label, value> in its option field, so to connect this data to a dropdown, we need to transform the data in the dropdown options property.

7. Mongo Query Syntax

Present  - The appsmith uses the database command syntax to query MongoDb
Suggestion - Appsmith uses the database command syntax to query MongoDB

8. Taking Inputs from Widgets

Present  - In the above example statusDropdown is the name of the widget and can be edited in the property pane by clicking on it
Suggestion - In the above example, statusDropdown is the name of the widget and can be edited in the property pane by clicking on it

Present  - The binding must be added inside quotes for it to be a valid JSON in the post body. See example below
Suggestion - The binding must be added inside quotes for it to be a valid JSON in the post body. See an example below

9. Connecting UI & Logic

Present  -Javascript can be used almost anywhere in appsmith inside {{ }}. The value evaluated inside the mustache is substituted in the field it is written in. This allows you to creatively configure different parts of your application.
Suggestion -Javascript can be used almost anywhere in Appsmith inside {{ }}. The value evaluated inside the mustache is substituted in the field it is written in. This allows you to creatively configure different parts of your application.

Present  - Similarly Logic can be added to this example to display something if no row is selected
Suggestion - Similarly, Logic can be added to this example to display something if no row is selected

10. Access Control

Present  - External users cannot access applications built on appsmith unless they are invited to the organisation
Suggestion - External users cannot access applications built on Appsmith unless they are invited to the organisation

11. Customer Support Dashboard

Present  - This tutorial explains how you can build a customer support dashboard in appsmith and covers various concepts from writing JS code to working with APIs
Suggestion - This tutorial explains how you can build a customer support dashboard in Appsmith and covers various concepts from writing JS code to working with APIs

12. Input

Present  - An input text field is used to capture a users textual input such as their names, numbers, emails etc. Inputs can be used in a form and can have custom validations.
Suggestion -An input text field is used to capture a user's textual input such as their names, numbers, emails etc. Inputs can be used in a form and can have custom validations.

Present  - The same can be used to filter data on the client side by writing a filter function using the text property
Suggestion -  Same can be used to filter data on the client-side by writing a filter function using the text property

13. Dropdown

This is an array of values of the options that are displayed in a multi select dropdown. This value changes if the default values of the dropdown change or the user changes an option selection

Lets you control whether you want the dropdown to be a Single Select or a Multi Select dropdown. Single Select allows users to choose only one value from the list of options whereas Multi Select allows for selection of more than one value by the user. 

Sets a default option that will be captured as user input unless it is changed by the user. Multiple values can be provided as CSV or an array of value string for a multi select dropdown.

Generic suggestion: multi-select in place of multi select in above 4 places it's used.

14. Datepicker

Present  - Datepicker is used to capture the date and time from a user. It can be used to filter data base on the input date range as well as to capture personal information such as date of birth.
Suggestion - Datepicker is used to capture the date and time of a user. It can be used to filter database on the input date range as well as to capture personal information such as date of birth.

15. Filepicker

Present - This property specifies the urls of the uploaded files that the file upload API returned. You can bind it to the API using {{ apiName.data.fileUrl }}
Suggestion - This property specifies the URLs of the uploaded files that the file upload API returned. You can bind it to the API using {{ apiName.data.fileUrl }}

16. Chart

Present - Displays a chart based on an array of objects with x and y values. You can transform the data from an api using a map 
Suggestion - Displays a chart based on an array of objects with X and Y values. You can transform the data from an API using a map 

17. Text

Present - The text widget also renders objects as JSOn by default
Suggestion - The text widget also renders objects as JSON by default

18. Container 

Present - Containers are used to group widgets together to form logical higher order widgets. Containers let you organize your page better and move all the widgets inside them together.
Suggestion - Containers are used to group widgets together to form logical higher-order widgets. Containers let you organize your page better and move all the widgets inside them together.

A few generic suggestions:
1. Appsmith in place of appsmith 
2. Full stop - either place it everywhere or nowhere. It's almost at half the places.

**Comment by AninditaBasu** [comment]
@Saket2 @Nikhil-Nandagopal Is this issue already assigned to someone? If not, I'd like to take it up for Hacktoberfest.

**Comment by Nikhil-Nandagopal** [comment]
@AninditaBasu this hasn't been assigned yet, if no one picks it by hacktoberfest, it's yours!

**Comment by AninditaBasu** [comment]
Cool! Calling dibs, then.

**Comment by Saket2** [comment]
@AninditaBasu This is done. PR : https://github.com/appsmithorg/appsmith-docs/pull/6

**Comment by ShyamPraveenSingh** [comment]
Hi, I would like to work on this.

**Comment by Saket2** [comment]
PR has been raised for this : https://github.com/appsmithorg/appsmith-docs/pull/15

**Comment by mohanarpit** [comment]
@ShyamPraveenSingh Welcome to Appsmith & thank you for expressing interest in contributing. Unfortunately, this issue has already been claimed by another user. 

Please go through our issue list [here](https://github.com/appsmithorg/appsmith/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22) & ask to work on another issue.

**Comment by ShyamPraveenSingh** [comment]
Sure thanks.
